packages/flutter/test/cupertino/refresh_test.dart

```diff
@@ -124,15 +124,15 @@ void main() {
       verify(mockHelper.builder(
         any,
         RefreshIndicatorMode.drag,
-        50.0,
+        32.08099300425676,
```

Comment:
For future reference (no need to change anything in this patch): It's generally a bad sign when you have a non-round number like this in a test. Normally, the numbers in a test should have obvious sources: e.g. they should be computed on the fly, or they should be round numbers that can obviously be derived from other round numbers, etc.

The problem with a number like this one is that as a reviewer I have no idea if it's correct. Is it 32.08099300425676? Or should it be 32.08099310425676? Or 32.08099300425672? There's no real way for me to know. This means that when the test is changed again, I've no way to know if the test was originally wrong, or if the person updating the test made a mistake.

Reply:
see also https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#avoid-mysterious-and-magical-numbers-that-lack-a-clear-derivation which I just added